lib/validate.js

```diff
@@ -49,12 +50,12 @@ module.exports = function makeValidatorMiddleware (middleware, schema, opts) {
           reqSchema.properties.query.properties[p.name] = p.schema
           p.required && !reqSchema.properties.query.required.includes(p.name) && reqSchema.properties.query.required.push(p.name)
           break
-        case 'header':
-          // eslint-disable-next-line no-case-declarations
```

Comment:
Please don't change formatting for no purpose in a feature PR.

Reply:
Removed related commit